coral/src/app/features/topics/acl-request/TopicAclRequest.test.tsx

```diff
@@ -776,7 +775,6 @@ describe("<TopicAclRequest />", () => {
         name: "Consumer group *",
       });
       const tooLong = new Array(152).join("a");
-      console.log(tooLong);
```

Comment:
🙈

Reply:
Hihi. We may need a linting rule for that, too, I tend to forget that sometimes, too :D